networks/dev/blockchain-dev/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+##
+## Input parameters
+##
+export PIO_HOME="/provenance/nodedev"
+BINARY=/usr/bin/${BINARY:-provenanced}
+LOG=${LOG:-provenance.log}
+
+##
+## Generate genesis if not already present
+##
+if ! [ -f ${PIO_HOME}/config/genesis.json ]; then
+  "${BINARY}" -t --home "${PIO_HOME}" init --chain-id=chain-dev testing ; \
```

Comment:
The ` ; \` line endings aren't needed here. They don't actually hurt anything, though.